src/categories/create.js

```diff
@@ -39,15 +40,18 @@ module.exports = function(Categories) {
 
 				var defaultPrivileges = ['find', 'read', 'topics:create', 'topics:reply'];
 
-				async.series([
```

Comment:
Is there a reason this is wrapped around the `async.series` instead of being one of the function calls inside?